Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:16
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package.json package-lock.json ./
```

Comment:
Incorrect comment. Wildcard not used.

Reply:
You're right, that was retrieved from another projects and not adjusted.